build/Packages.props

```diff
@@ -2,6 +2,8 @@
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
   <PropertyGroup>
+    <CakeScriptinTransportVersion>0.1.0-unstable0091</CakeScriptinTransportVersion>
+    <DotnetScriptNuGetMetadataResolver>2.0.3</DotnetScriptNuGetMetadataResolver>
```

Comment:
this is not needed, we removed it recently - is this a rebase issue?

Reply:
Might have been introduced when @david-driscoll just rebased.

Reply:
Might be my changes. 🐑